core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -125,7 +127,11 @@ private[storage] class BlockManagerDecommissioner(
                   logDebug(s"Migrated sub-block $blockId")
                 }
               }
-              logInfo(s"Migrated $shuffleBlockInfo to $peer")
+              val endTime = System.nanoTime()
```

Comment:
Why do we need take `nanoTime()` then convert it `toMillis`?  What is the problem of directly getting `System.currentTimeMillis()` for calculation?